Source/Game.cs

```diff
@@ -60,10 +61,12 @@ private enum TransitionStep
 	private readonly FMOD.Studio.EVENT_CALLBACK audioEventCallback;
 	private int audioBeatCounter;
 	private bool audioBeatCounterEvent;
+    
+    private ImGuiManager imGuiManager;

```

Comment:
I get that, but it's still inconsistant in the rest of the project.
Imo the only real solutíon would be to setup an `.editorconfig` and require every contributor to use that one.
That would ofc cause probably big issues with merging upstream, but it seems like commits are slowing down so that might not actually be that big of a problem.

Reply:
yeah, I wasn't enforcing that at first, but I'm realizing not doing it makes it harder to keep things in sync with the upstream, since they mostly do use tabs, so I probably want to start being a bit more careful about it going forward. A lot of the cases that it is inconsistent are from before I really realized that it was an issue.

I won't hold it back from blocking this PR, but just was mostly giving you a heads up to keep an eye on it if you make future PRs. I might end up going through and cleaning up a lot of spots where it's already happened at some point though. Same thing with the curly brace issue.